code/modules/sanity/sanity_mob.dm

```diff
@@ -52,14 +52,14 @@
 	var/level_change = 0
 
 	var/insight
-	var/max_insight = INFINITY
-	var/insight_passive_gain_multiplier = 1
+	var/max_insight = 1
```

Comment:
![image](https://user-images.githubusercontent.com/65828539/194624679-d4aeab96-2bf2-4073-b69a-674548c41936.png)

![image](https://user-images.githubusercontent.com/65828539/194624796-7d5cae3b-cf2a-4623-85f5-a835550a0dcf.png)

:thonker:

Reply:
![image](https://user-images.githubusercontent.com/112724221/194625082-ce9d23ee-449e-499c-a477-83ee9a893fcb.png)
:unthonker:

Reply:
Player would need insight 100 or more, but they wouldn't be able to get it higher than 1.

Reply:
wait

Reply:
FUCK

Reply:
![image](https://user-images.githubusercontent.com/65828539/194626080-b8ac5a30-1b49-448e-96ba-5b9895fe062a.png)

As good as `insight = 1`

Reply:
you've gaslighted me

Reply:
No?